public/styles.css

```diff
@@ -0,0 +1,83 @@
+html {
+    box-sizing: border-box;
+  }
+
+.items {
+    display: flex;
+    justify-content: center;
+    flex-wrap: nowrap;
+    min-width: 0px;
+    max-width: 1400px;
+    margin: auto;
+    flex-shrink: 2;
+}
+
+.related-allItems {
+    display: grid;
+    grid-template-columns: 50%, 50%;
+    grid-template-rows: 15%, 85%;
+}
+
+.related-fromShop {
+    grid-column: 1 / 2;
+    grid-row: 2 / 3;
+    display: grid;
+    grid-template-columns: 50% 50%;
+    grid-template-rows: 50% 50%;
+    justify-content: end;
+    padding-right: 30px;
+}
+
+
+
+.related-items {
+    grid-column: 2 / 3;
+    grid-row: 2 / 3;
+    display: grid;
+    grid-template-columns: 50% 50%;
+    grid-template-rows: 50% 50%;
+    justify-content: start;
+    padding-left: 30px;
+}
+
+.related-fromShopHeader {
+    grid-column: 1 / 2;
+    grid-row: 1/ 2;
+    display: flex;
+    justify-content: flex-start;
+    align-self: flex-end;
+    margin: 0;
+
+}
+
+.related-relatedHeader {
+    grid-column: 2 / 3;
+    grid-row: 1 / 2;
+    display: flex;
+    justify-content: flex-start;
+    padding-left: 45px;
+    align-self: flex-end;
+    margin: 0;
+}
+
+.related-shopIndividualItem {
+    padding: 9px;
+    display: flex;
+    flex-shrink: 1;
+}
+
+.related-relatedIndividualItem{
+    padding: 9px;
+    display: flex;
+    flex-shrink: 1;
+}
+
+
+img {
+    display: flex;
+    min-width: 150px;
+    min-height: 100px;
+    max-width: 300px;
+    max-height: 250px;
+    border-radius: 9px;
+}
```

Comment:
I'm not sure if it will work, but try changing the width and height to a % to see if it will auto size.